content/docs/components/hyperparameter-tuning/katib-config.md

```diff
@@ -114,7 +114,13 @@ All of these settings except **`image`** can be omitted. If you don't specify an
 
 1. `image` - Docker image name for the `random` suggestion.
 
-    **Must be specified**.
+    **Must be specified**. You can specify following images:
+
+    - `suggestion-chocolate`: [chocolate](https://github.com/AIworx-Labs/chocolate) based suggestion service which supports `grid`, `chocolate-random`, `chocolate-quasirandom`, `chocolate-bayesian-optimization` and `chocolate-mocmaes`.
+    - `suggestion-goptuna`: [Goptuna](https://github.com/c-bata/goptuna) based suggestion service which supports `cmaes`, `tpe` and `random`.
+    - `suggestion-hyperband`: [HpBandSter](https://github.com/automl/HpBandSter) based suggestion service which supports `hyperband`.
```

Comment:
I believe for hyperband we are using our own implementation. We can add link to Katib hyperband source code here.

Reply:
Thank you for your comment. Hm, we may change the file path of hyperband source code at https://github.com/kubeflow/katib/issues/1144.

Reply:
Yes, we will change the file path for hyperband. In this PR maybe just add link to hyperband service: https://github.com/kubeflow/katib/blob/master/pkg/suggestion/v1alpha3/hyperband_service.py?